src/amo/components/EditableCollectionAddon/styles.scss

```diff
@@ -7,17 +7,36 @@
 .EditableCollectionAddon {
   display: grid;
   grid-gap: 24px;
-  grid-template-columns: 32px auto 32px 64px;
+  grid-template-columns: 32px repeat(auto-fill, minmax(80px, 90px)) 32px 64px;
```

Comment:
Why do we use `repeat` with `auto-fill` when there's only ever one column we're dealing with?